.gitignore

```diff
@@ -5,6 +5,7 @@ __pycache__/
 *.dist-info/
 .mypy_cache/
 .benchmarks/
+.python-version
```

Comment:
I'm using a pyenv virtualenv for the repo, so adding this to ignore my virtual env version